pkg/providers/kubernetes/README.md

```diff
@@ -73,7 +73,7 @@ By default, the vCluster Helm chart assumes the `ingress-nginx` controller is in
 This can be installed in a similar manner to the following:
 
 ```shell
-helm upgrade --install nginx-ingress nginx/ingress-nginx -n ingress-nginx --create-namespace --set controller.ingressClassResource.default=true --set 'controller.extraArgs.enable-ssl-passthrough='
+helm upgrade --install nginx-ingress nginx/nginx-ingress -n ingress-nginx --create-namespace --set controller.ingressClassResource.default=true --set 'controller.extraArgs.enable-ssl-passthrough='
```

Comment:
Oh hell, that must have been copied from an archaic bit of cruft on my disk. It should definitely be ingress-nginx, the Kubernetes one https://artifacthub.io/packages/helm/ingress-nginx/ingress-nginx

Reply:
https://github.com/unikorn-cloud/kubernetes gives the instruction

     helm repo add nginx https://helm.nginx.com/stable